src/main/java/medic/gateway/alert/Db.java

```diff
@@ -152,8 +153,9 @@ public void onCreate(SQLiteDatabase db) {
 					"%s TEXT, " +
 					"%s INTEGER NOT NULL, " +
 					"%s TEXT NOT NULL, " +
-					"%s TEXT NOT NULL)",
-				tblWO_MESSAGE, WOM_clmID, WOM_clmSTATUS, WOM_clmFAILURE_REASON, WOM_clmLAST_ACTION, WOM_clmTO, WOM_clmCONTENT));
+					"%s TEXT NOT NULL, " +
+					"%s INTEGER NOT NULL)",
+				tblWO_MESSAGE, WOM_clmID, WOM_clmSTATUS, WOM_clmFAILURE_REASON, WOM_clmLAST_ACTION, WOM_clmTO, WOM_clmCONTENT, WOM_clmRETRIES));
```

Comment:
I think you'll need to update the `onUpgrade` function to add this column to existing dbs.

Reply:
I'm going to do this in previous PR related to retries and let you know, so basically:
1. Change the schema version from 6 to 7
2. Move the table creation to a migration method
3. Add hardcoded value in the new column, if the DB is not clean (? not sure if this makes sense)
4. Call the migration method in the onCreate and onUpgrade methods as done with other migrations. 
🤔

Reply:
Sounds good. Yes set a default of `0` maybe?

Reply:
Done!